go/worker/keymanager/handler.go

```diff
@@ -48,16 +48,12 @@ func (h *hostHandler) Handle(ctx context.Context, body *protocol.Body) (*protoco
 		switch body.HostRPCCallRequest.Endpoint {
 		case runtimeKeymanager.EnclaveRPCEndpoint:
 			// Call into the remote key manager.
-			rsp, pf, err := h.remoteClient.CallEnclave(ctx, &p2p.CallEnclaveRequest{
-				Data: body.HostRPCCallRequest.Request,
-			})
+			rsp, err := h.remoteClient.CallEnclave(ctx, body.HostRPCCallRequest.Request, body.HostRPCCallRequest.PeerFeedback)
 			if err != nil {
 				return nil, err
 			}
-			// TODO: Support reporting peer feedback from the enclave.
-			pf.RecordSuccess()
 			return &protocol.Body{HostRPCCallResponse: &protocol.HostRPCCallResponse{
-				Response: cbor.FixSliceForSerde(rsp.Data),
+				Response: cbor.FixSliceForSerde(rsp),
```

Comment:
Btw is this `FixSliceForSerde` still needed? `serde` is not used anymore for cbor serialization, right?

Reply:
True, we could fix it in oasis-cbor, but currently it also doesn't allow deserialization into vectors from NULL, so it is still needed. Will open an issue for it though.